ts/utils/faq.ts

```diff
@@ -67,9 +68,11 @@ export const getFAQsFromCategories = (
   );
 
   return Array.from(faqIDs).map<FAQType>(id => {
+    const title = `faq.${id}.title` as TranslationKeys;
+    const content = `faq.${id}.content` as TranslationKeys;
     return {
```

Comment:
you could use options as second parameter instead force casting the key
```suggestion
    return {
      title: I18n.t(`faq.${id}.title`, { defaultValue: "faq title n/a" }),
      content: I18n.t(`faq.${id}.content`, { defaultValue: "faq content n/a" })
    };
```